roles/vdm/templates/kustomization.yaml

```diff
@@ -25,4 +25,12 @@ namespace: {{ NAMESPACE }}
 {% endif %}
 
 {% endfor %}
-
+{% if "components" not in ordered_overlays.result %}
```

Comment:
This update to the kustomization.yaml template handles the scenario when DAC does not contribute any hard-coded component entries into ordered_overlays (as when TLS is disabled) and then skips over adding user submitted components under site-config such as SingleStore.